src/components/templates/project-wrapper.vue

```diff
@@ -41,7 +41,7 @@ import MainTitle from '../atoms/common/main-title';
 import ProjectContent from '../organisms/project/project-content';
 import ProjectContentFooter from '../molecules/project/project-content-footer';
 
-import { projects } from '../static/data/projects';
+import { project } from '../static/data/index.js';
```

Comment:
```suggestion
import { project } from '../static/data';
```
마찬가지로 index.js 를 숨기는 것을 제안드립니다.